apps/web/src/components/reporting/DataExtractReport.tsx

```diff
@@ -81,7 +81,7 @@ export const DataExtractReport = () => {
                   label='End Date'
                   format='yyyy-MM-dd'
                   min={dayjs(values.from || MIN_DATE).toDate()}
-                  max={dayjs(values.to || getMaxDate()).toDate()}
+                  max={dayjs(getMaxDate() || values.to).toDate()}
```

Comment:
I don't remember why I needed `values.to` condition.

Reply:
I'll try removing it and see what happens

Reply:
getMaxDate always returns a value, so we don't need it.